code/modules/capture_the_flag/ctf_classes.dm

```diff
@@ -0,0 +1,228 @@
+// GENERIC CLASSES
+
+/datum/outfit/ctf
+	name = "CTF Rifleman (Solo)"
+	ears = /obj/item/radio/headset
+	uniform = /obj/item/clothing/under/syndicate
+	suit = /obj/item/clothing/suit/space/hardsuit/shielded/ctf
+	toggle_helmet = FALSE // see the whites of their eyes
+	shoes = /obj/item/clothing/shoes/combat
+	gloves = /obj/item/clothing/gloves/combat
+	id = /obj/item/card/id/away
+	belt = /obj/item/gun/ballistic/automatic/pistol/deagle/ctf
+	l_pocket = /obj/item/ammo_box/magazine/recharge/ctf
+	r_pocket = /obj/item/ammo_box/magazine/recharge/ctf
+	r_hand = /obj/item/gun/ballistic/automatic/laser/ctf
+
+	///Description to be shown in the class selection menu
+	var/class_description = "General purpose combat class. Armed with a laser rifle and backup pistol."
+	///Radio frequency to assign players with this outfit
+	var/team_radio_freq = FREQ_COMMON // they won't be able to use this on the centcom z-level, so ffa players cannot use radio
+	///Icon file for the class radial menu icons
+	var/icon = 'icons/hud/radial_ctf.dmi'
+	///Icon state for this class
+	var/icon_state = "ctf_rifleman"
+	///Do they get a headset?
+	var/has_radio = TRUE
+	///Do they get an ID?
+	var/has_card = TRUE
+	///Which slots to apply TRAIT_NODROP to the items in
+	var/list/nodrop_slots = list(ITEM_SLOT_OCLOTHING, ITEM_SLOT_GLOVES, ITEM_SLOT_FEET, ITEM_SLOT_ICLOTHING, ITEM_SLOT_EARS)
+
+/datum/outfit/ctf/post_equip(mob/living/carbon/human/H, visualsOnly=FALSE)
```

Comment:
please make this a descriptive variable as single letter variables are against our contributing guidelines and are cringe